llama-index-integrations/llms/llama-index-llms-openai/llama_index/llms/openai/utils.py

```diff
@@ -229,6 +229,12 @@ def is_chat_model(model: str) -> bool:
 
 
 def is_function_calling_model(model: str) -> bool:
+    # checking whether the model is fine-tuned or not.
+    if model.startswith("ft-"):  # legacy fine-tuning
```

Comment:
nit: can we leave a comment here of an example of what the full model name looks like and what parts of it we're trying to keep in both cases?

Reply:
fyi it looks something like `ft:gpt-3.5-turbo:acemeco:suffix:abc123` -- the legacy fine-tuned format isn't on openai docs anymore, but we have a similar check elsewhere